AdyenUI/UI/Form/Items/Text/FormTextItemStyle.swift

```diff
@@ -46,7 +46,7 @@ public struct FormTextItemStyle: FormValueItemStyle {
     public var backgroundColor: UIColor = .clear
     
     /// The accent color for warnings and error messages.
-    public var errorColor = UIColor.Adyen.defaultRed
+    public var errorColor = AdyenTheme().currentColorScheme.destructive
```

Comment:
I know `AdyenTheme` is not injected yet. Instead of creating an instance every time it is used, lets create an `adyenTheme` property so at least it can easily be replaced in the future when you inject it.